libs/blocks/media/media.js

```diff
@@ -30,9 +30,10 @@ function decorateAvatar(el) {
 function decorateQr(el) {
   const text = el.querySelector('.text');
   if (!text) return;
-  const appStore = text.children[(text.children.length - 1)];
```

Comment:
Alternatively, we could add `.querySelector('a')`, since the structure should be set in stone.

Reply:
Yeah this seems to be hardcoded, so can do that